watchdog/main.go

```diff
@@ -271,6 +299,7 @@ func main() {
 	}
 
 	http.HandleFunc("/", makeRequestHandler(&config))
+	http.HandleFunc("/health", health())
```

Comment:
It should have been with a prefix that cannot clash with the function i.e. `/_/` - I think this was mentioned in the issue. Please can you change it?

Reply:
My bad .. I will add that.